src/utils/data-title-utils.js

```diff
@@ -106,7 +106,7 @@ export function getMeasurePopoverData({
 export function getMeasureTitleData(meaInfo, index) {
   return {
     breadcrumbs: [],
-    drilldown: false,
+    DRILL_DOWN: false,
```

Comment:
I can see why you changed from `DRILLDOWN` -> `DRILL_DOWN` in `src/picasso-components/react-components/icons/index.js`, since it can be considered a "constant"... But in other places, like here, or in the translations, it should not be changed into capital letters... maybe a search and replace mistake?

Reply:
yes, this is a typo. Jingjing has told me and I have fixed it.